sinatra/skillsengine.rb

```diff
@@ -0,0 +1,36 @@
+require 'rest-client'
+require 'json'
+
+module SkillsEngine
+  @@client_id = ENV['SKILLS_ENGINE_ID']
+  @@client_secret = ENV['SKILLS_ENGINE_SECRET']
+  @@access_token = nil
+
+  private_class_method def self.get_access_token()
```

Comment:
The style is to not include parentheses at the end of method calls or definitions if there's no argument being passed. They're not necessary.